tpot/metrics.py

```diff
@@ -45,10 +45,12 @@ def balanced_accuracy(y_true, y_pred):
     all_classes = list(set(np.append(y_true, y_pred)))
     all_class_accuracies = []
     for this_class in all_classes:
-        this_class_sensitivity = \
-            float(sum((y_pred == this_class) & (y_true == this_class))) /\
-            float(sum((y_true == this_class)))
-
+        if float(sum((y_true == this_class))) == 0:
+            this_class_sensitivity = 0
+        else:
+            this_class_sensitivity = \
+                float(sum((y_pred == this_class) & (y_true == this_class))) /\
+                float(sum((y_true == this_class)))
         this_class_specificity = \
             float(sum((y_pred != this_class) & (y_true != this_class))) /\
             float(sum((y_true != this_class)))
```

Comment:
The specificity calculation should be encapsulated in this `if` statement as well, right?